packages/create-snowpack-app/index.js

```diff
@@ -151,9 +157,25 @@ if (requiredVersion < 10) {
     cwd: targetDirectory,
     stdio: "inherit",
   };
-  const npmInstallProcess = useYarn
-    ? execa("yarn", ["--silent"], npmInstallOptions)
-    : execa("npm", ["install", "--loglevel", "error"], npmInstallOptions);
+
+  function installProcess(packageManager) {
+    switch (packageManager) {
+      case "npm":
+        return execa(
+          "npm",
+          ["install", "--loglevel", "error"],
+          npmInstallOptions
+        );
+      case "yarn":
+        return execa("yarn", ["--silent"], npmInstallOptions);
+      case "pnpm":
+        return execa("pnpm", ["install"], npmInstallOptions);
```

Comment:
Is there a similar "--silent" flag that we can use here?

Reply:
It can use `--reporter=silent` https://pnpm.js.org/en/cli/install#reporter-lt-name

Reply:
I have added the silent flag. I also updated my system from pnpm v4 to the latest v5.2.6 and re-run the tests:
```
NPM

Test Suites: 1 passed, 1 total
Tests:       9 passed, 9 total
Snapshots:   0 total
Time:        100.897 s

PNPM

Test Suites: 1 passed, 1 total
Tests:       9 passed, 9 total
Snapshots:   0 total
Time:        1.918 s, estimated 101 s
```
pnpm seems insanely faster :) You had best double check all is well before merging, but it looks good to me :)